workflow-templates/repo-health-workflow.yml

```diff
@@ -0,0 +1,25 @@
+name: Repo Health Workflow Call
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+    inputs:
+      ORG_NAMES: 
+        description: "Space separated list of organisation names to parse repos. i.e. 'openedx edx . . . ' "
+        type: string
+        default: "edx openedx"
+
+jobs:
+  call-repo-health-job-workflow:
+    uses: usamasadiq/openedx-github/.github/workflows/repo-health-job.yml@master
+    with:
+      ORG_NAMES: ${{ inputs.ORG_NAMES }}
+      REPORT_DATE: ""
+      ONLY_CHECK_THIS_REPOSITORY: ""
+      REPO_HEALTH_GOOGLE_CREDS_FILE: ""
+      REPO_HEALTH_REPOS_WORKSHEET_ID: 0
+      REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL: ""
```

Comment:
It feels like all of this stuff for getting data from or updating in Google Sheets should be in the 2U private workflow call, rather than the shared workflow.

Reply:
This logic for getting and updating data in sheets was copied to provide this feature to other orgs as well. Since the values of these parameters will be coming from secrets, other orgs can just save their own credentials in secrets to avail the feature of populating and maintaining their own repo health sheet. 

If we want to keep this feature only within 2U, then we'll need to create a seperate workflow for 2U org. 

Right now, the whole shared workflow/script is exactly the same as the jenkins job and different orgs can provide their own paths/secrets to store their data on their desired paths/repos.

Reply:
I guess this is ok as long as the Google Sheet updating is gracefully skipped if no data is provided.  I want to make sure that having a Google Workplace account isn't a prerequisite for running the repo health workflow.

Reply:
We've tried it when testing the workflow from the forked repo, if no google sheet credentials are provided and the workflow is called with default values, then the step for updating google sheet is automatically skipped.